openverse_catalog/dags/common/constants.py

```diff
@@ -22,7 +22,9 @@
     "email_on_failure": False,
     "email_on_retry": False,
     "email": [CONTACT_EMAIL],
-    "retries": 2,
+    "retries": os.getenv(
+        "DEFAULT_RETRY_COUNT"
+    ),  # Default retries is set to 2 in .env file,
```

Comment:
In the case where this environment variable isn't set, we'd still like this to default to 2. We can change change that by doing:
```suggestion
    "retries": os.getenv("DEFAULT_RETRY_COUNT", 2),
```